executor/analyze.go

```diff
@@ -104,6 +104,10 @@ func (e *AnalyzeExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	}
 	close(taskCh)
 	statsHandle := domain.GetDomain(e.ctx).StatsHandle()
+	historicalStatsEnabled, err := statsHandle.CheckHistoricalStatsEnable()
+	if err != nil {
+		logutil.BgLogger().Error("Check TiDBEnableHistoricalStats Failed.")
+	}
```

Comment:
Please move these lines to L185 since we don't need to check it if some other errors occur.